### PR TITLE
[Remove FontAwesome] Remove from `BottomBar`, `LeftNav`; remove some excessive styling from `BottomBar` and `BottomBarWeb`

### PR DESCRIPTION
**Pull request by haileyok**
## Why

We would like to remove the FontAwesome icon library from the app, as it is currently the biggest library that we are using in the app. This would shave about 1 MB off of our bundle size (parsed), which would be a significant improvement. This PR replaces the FontAwesome icons that are used in the `LeftNav` and the `BottomBar` with icons from the new icon library.

Additionally, while working on the `BottomBar` replacements, I realized that there was some excessive styling that could be achieved with flexbox rather than margins, negative margins, etc. I reworked these aspects of the styles here.

Note that these icons are _not_ a part of FontAwesome, but the natural first step in working through these feels like replacing the main navigation icons, which is why I started here.

## Test Plan

The icons that have changed here are on the left nav and the bottom bar. I've gone through all of these icons to ensure that they properly switch between filled/not filled when the respective tab is selected. I have also verified that the updated styles work on 

The more important thing that we will want to test here is if we _like_ the icons. Some of the icons do not have 1:1 replacements as far as design goes, so there are some subtle differences between them - primarily the home icon and the alert/bell icon. If we feel strongly about a particular icon not looking great, then we might want to replace.

### Android

Screenshot 2024-04-06 at 5 28 38 PM

### iOS

Screenshot 2024-04-06 at 5 28 50 PM

### Web Desktop

Screenshot 2024-04-06 at 5 29 07 PM

### Web Mobile

Screenshot 2024-04-06 at 5 29 47 PM

### CODE REVIEW

**Comment by haileyok** [comment]
Note that the bottom bar on web being slightly offset to the right is due to the ghost scroll gutter issue that has not yet been fixed.

File: src/view/shell/desktop/LeftNav.tsx [hunk 1]
```diff
@@ -299,141 +319,57 @@ export function DesktopLeftNav() {
...
           <NavItem
             href="/lists"
-            icon={
-              <ListIcon
-                style={pal.text}
-                size={isDesktop ? 26 : 30}
-                strokeWidth={2}
-              />
-            }
+            icon={<Bars style={t.atoms.text} size="lg2" />}
```
Comment (mozzius) [inline src/view/shell/desktop/LeftNav.tsx:351]: I don't love this icon, but I don't see anything better in figma

**Review by mozzius — APPROVED** [review]
The icons look a little bit heavier than the current ones, however I don't think that's an issue. Tested iOS sim + web, LGTM

**Comment by haileyok** [comment]
Yea I wanted to see what @estrattonbailey thought on that. Honestly, we don't need to get rid of these icons - they are not using FontAwesome anyway - if we don't like them. Really was just playing around w/ stuff on the weekend. With that said I'd just move the old icons into using the same system as we are using now if we wanted to keep them.

File: src/components/icons/TEMPLATE.tsx [hunk 2]
```diff
@@ -28,7 +28,14 @@ export const IconTemplate_Stroke2_Corner0_Rounded = React.forwardRef(
...
 
-export function createSinglePathSVG({path}: {path: string}) {
+export function createSinglePathSVG({
+  path,
+  strokeWidth = 0,
+}: {
+  path: string
+  strokeWidth?: number
+}) {
+  console.log(strokeWidth)
```
Comment (estrattonbailey) [inline src/components/icons/TEMPLATE.tsx:38]: Log :P

File: src/components/icons/Sample.tsx
Comment (estrattonbailey) [inline src/components/icons/Sample.tsx:1]: Nice nice

File: src/view/shell/bottom-bar/BottomBarWeb.tsx [hunk 3]
```diff
@@ -53,68 +63,47 @@ export function BottomBarWeb() {
...
               <NavItem routeName="Feeds" href="/feeds">
                 {({isActive}) => {
-                  return (
-                    <HashtagIcon
-                      size={22}
-                      style={[styles.ctrlIcon, pal.text, styles.feedsIcon]}
-                      strokeWidth={isActive ? 4 : 2.5}
-                    />
-                  )
+                  const Icon = isActive ? Hashtag : HashtagFilled
```
Comment (estrattonbailey) [inline src/view/shell/bottom-bar/BottomBarWeb.tsx:99]: Should this be switched around?

**Review by estrattonbailey — APPROVED** [review]
Looks great!

**Comment by haileyok** [comment]
We did it!

**Pull request closed without merging**